### PR TITLE
[80] Implement actor apply jobs API @POST

### PR DESCRIPTION
**Pull request by palmcm**
## What did you do

-add endpoint for applyJob

## Demo

- login as actor 
- check exist resume or create new one
- try to test apply job

### CODE REVIEW

File: apps/api/src/modules/job/application/application.service.spec.ts [hunk 1]
```diff
@@ -81,4 +90,90 @@ describe('ApplicationService', () => {
...
+      jest
+        .spyOn(resumeRepository, 'getResumeById')
+        .mockResolvedValue(MOCK_RESUME)
+      jest.spyOn(repository, 'getApplicationbyActorJob').mockResolvedValue(null)
+    })
+
+    it('should apply by job id correctly', async () => {
+      expect(
+        service.applyJob(MOCK_JOB_ID, MOCK_RESUME.resumeId, MOCK_ACTORS_ID),
+      ).resolves.toEqual({ applicationId: MOCK_APPLICATION.applicationId })
```
Comment (miello) [inline apps/api/src/modules/job/application/application.service.spec.ts:129]:
```suggestion
      await expect(
        service.applyJob(MOCK_JOB_ID, MOCK_RESUME.resumeId, MOCK_ACTORS_ID),
      ).resolves.toEqual({ applicationId: MOCK_APPLICATION.applicationId })
```

Comment (miello) [reply apps/api/src/modules/job/application/application.service.spec.ts:129]: Please add `await` for all resolves or rejects checking

File: apps/api/src/modules/job/application/application.controller.ts [hunk 2]
```diff
@@ -36,4 +43,19 @@ export class ApplicationController {
...
+  @ApiBadRequestResponse({ description: 'Resume not found' })
+  @ApiForbiddenResponse({ description: 'User is not Actor or job is not open' })
+  @ApiConflictResponse({ description: 'User already apply to the job' })
+  applyJob(
+    @Param('id') id: string,
+    @Body() body: ResumeIdDto,
+    @User() user: JwtDto,
+  ) {
+    return this.applicationService.applyJob(+id, body.resumeId, user.userId)
+  }
```
Comment (miello) [inline apps/api/src/modules/job/application/application.controller.ts:60]: Where is success response in swagger details ?

**Review by miello — CHANGES_REQUESTED** [review]

**Comment by palmcm** [comment]
@miello pls review ;w;

**Review by miello — APPROVED** [review]
I'm sincerely apologize for late review

**Pull request merged**